stix_shifter_modules/alertflex/configuration/config.json

```diff
@@ -27,14 +27,14 @@
         }
     },
     "configuration": {
-        "type" : "fields",
         "auth": {
-            "username": {
+            "type" : "fields",
+            "login": {
```

Comment:
If username is changing to login, you will also need to change it here as well https://github.com/opencybersecurityalliance/stix-shifter/blob/develop/stix_shifter_modules/alertflex/configuration/lang_en.json#L27.